### ISSUE TITLE
100% lighthouse and semantic HTML

### ISSUE THREAD

**Issue by kirch1** [kirch1/whats-cookin#85]
You must score as close to 100% as possible with the “Lighthouse Accessibility Audit”. Be prepared to explain any accessibility audits your application is failing.
Your HTML must be written semantically and should use ARIA tags (ONLY if needed / appropriate)
